### PR TITLE
read model running status from llm-inference server

### PR DESCRIPTION
**Pull request by Rader**
Plese note:

1. mode UI widget type default to ModelWidgetTypeGeneration, need to config later
2. need to set new env `STARHUB_SERVER_INFERENCE_SERVER_ADDR` for llm-inference server address

### CODE REVIEW

File: builder/inference/app.go [hunk 1]
```diff
@@ -1,7 +1,8 @@
 package inference
```
Comment (pulltheflower) [inline builder/inference/app.go:1]: Should we change the filename to `client.go`?

**Review by pulltheflower — COMMENTED** [review]
Just one question. Others LGTM

File: builder/inference/app.go (hunk 1 above)
Comment (Rader) [reply builder/inference/app.go:1]: sure

**Review by pulltheflower — APPROVED** [review]
LGTM

**Pull request merged**